### PR TITLE
Great split

### PR DESCRIPTION
**Pull request by olafurpg**
This PR creates three new sites.

- index.html - landing page
- tutorial/index.html - same tutorial as before but without macro annotations
- paradise/index.html - docs for macro annotations

### CODE REVIEW

**Comment by xeno-by** [comment]
> same tutorial as before but without macro annotations

Please have a special 👍 from me for this one.

File: readme/LandingPage.scalatex [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+              Feature
+            @th
+              Supported?
+        @tbody
+          @def feature(name: String, supported: String) = tr(td(name), td(supported))
+          @feature("Compiler messages", "Yes")
+          @feature("Symbol at position", "Yes")
+          @feature("Symbol signature", "Yes")
+          @feature("Type at position", "No")
+          @feature("Type members (completion)", "No")
```
Comment (xeno-by) [inline readme/LandingPage.scalatex]: Is this really equivalent to completion though?

File: readme/LandingPage.scalatex [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+          @feature("Macro expansions", "No")
+          @feature("Inferred type arguments", "Yes")
+          @feature("Implicit arguments", "Yes")
+          @feature("Implicit conversions", "Yes")
+          @feature("Inferred .apply/.unapply", "Yes")
```
Comment (xeno-by) [inline readme/LandingPage.scalatex]: Maybe we could add `@feature("Other desugarings", "Planned")`?

File: readme/LandingPage.scalatex [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+         @lnk("Metadoc", "http://scalameta.org/metadoc/"): online code browser
+         with "Jump to definition" and "See references".
+      @li
+        @lnk("Imclipitly", "https://github.com/ShaneDelmore/imclipitly"):
+        Scala-Clippy advice generator for implicit enrichments.
+      @li
+        @lnk("Stags", "https://github.com/pjrt/stags"): Scala tags generator.
+      @li
+        @lnk("AST explorer", "https://astexplorer.net/#/gist/22cf8a3fcb2155c087ae94b4d194c1b6/d10c646ecfae4c69c919408aa3aaefb2deda2df7"):
+        interactively explorer of Scala syntax.
```
Comment (xeno-by) [inline readme/LandingPage.scalatex]: interactive

File: readme/LandingPage.scalatex [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+    @p
+      @b
+        Semantic Tooling at Twitter
+      (ScalaDays Copenhagen 2017). This talk introduces semantic databases, the cornerstone of
+      the scalameta semantic API, and explains how semantic databases can be used to integrate
+      with Kythe, a language-agnostic ecosystem for developer tools. In this talk, we presented
+      our vision of next-generation semantic tooling for the Scala ecosystem.
+
+    @p
+      Video: coming soon
```
Comment (xeno-by) [inline readme/LandingPage.scalatex]: Let's remove the "coming soon" bit.

File: readme/tutorial/FAQ.scalatex [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  @sect{What is the quasiquote for X?}
+    Here is an overview of quasiquote syntax: @lnk("https://github.com/scalameta/scalameta/blob/master/notes/quasiquotes.md").
+
+  @sect{Can I use Scalameta with Scala.js?}
+    Yes, the main Scalameta modules support Scala.js.
+
+
+  @sect{I'd like to use the semantic api as a replacement for scalac's
+        presentation compiler, is this doable? intended usage?}
+    It's not doable and it's not the intended usage. The prime application we
```
Comment (xeno-by) [inline readme/tutorial/FAQ.scalatex]: "In principle, this should be doable, but it requires custom development and is not supported at the moment."?

File: readme/tutorial/FAQ.scalatex [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+      @li
+        @sect.ref{Installation}
+      @li
+        @lnk("sbt-semantic-example", "https://github.com/scalameta/sbt-semantic-example")
+      @li
+        @lnk("scalacenter/scalafix.g8", "https://scalacenter.github.io/scalafix/#scalacenter/scalafix.g8")
+
+  @sect{Does Scalameta integrate with Zinc in order to achieve the semantic api?}
+    No, Zinc and Scalameta and unrelated.  The Scalameta Semantic
+    API is enabled with the scalac compiler plugin @sect.ref{Scalahost}.
```
Comment (xeno-by) [inline readme/tutorial/FAQ.scalatex]: Shall we rename this section?

**Review by xeno-by — CHANGES_REQUESTED** [review]

File: readme/LandingPage.scalatex (hunk 1 above)
Comment (olafurpg) [reply readme/LandingPage.scalatex]: I added "Auto completion -> No" in a separate row.

File: readme/LandingPage.scalatex (hunk 2 above)
Comment (olafurpg) [reply readme/LandingPage.scalatex]: Other desugarings is too vague IMO, the table should contain concrete information.

File: readme/LandingPage.scalatex (hunk 3 above)
Comment (olafurpg) [reply readme/LandingPage.scalatex]: Done.

File: readme/LandingPage.scalatex (hunk 4 above)
Comment (olafurpg) [reply readme/LandingPage.scalatex]: Removed.

File: readme/tutorial/FAQ.scalatex (hunk 5 above)
Comment (olafurpg) [reply readme/tutorial/FAQ.scalatex]: 👍

File: readme/tutorial/FAQ.scalatex (hunk 6 above)
Comment (olafurpg) [reply readme/tutorial/FAQ.scalatex]: Agreed, I removed references to "scalahost".

**Review by olafurpg — COMMENTED** [review]
Thank you for the review @xeno-by !

**Comment by olafurpg** [comment]
Closing and reopening to trigger travis.

**Review by xeno-by — APPROVED** [review]

**Comment by olafurpg** [comment]
Let's see this go live :)

**Pull request merged**